bot/exts/easter/conversationstarters.py

```diff
@@ -3,13 +3,23 @@
 import random
 from pathlib import Path
 
+from discord import Embed
 from discord.ext import commands
 
+from bot.utils.decorators import override_in_channel
+
+
 log = logging.getLogger(__name__)
 
+
 with open(Path("bot/resources/easter/starter.json"), "r", encoding="utf8") as f:
     starters = json.load(f)
 
+with open(Path("bot/resources/easter/py_topics.json"), "r", encoding="utf8") as f:
+    # First ID is #python-general and the rest are top to bottom categories of Topical Chat/Help.
+    py_topics = json.load(f)["python-channels"]
+    all_python_channels = [int(channel_id) for channel_id in py_topics.keys()]
```

Comment:
Similarly here with the naming.
```suggestion
    PY_TOPICS = json.load(f)["python-channels"]
    ALL_PYTHON_CHANNELS = [int(channel_id) for channel_id in PY_TOPICS.keys()]
```